### ISSUE TITLE
Why isn't the loss backpropagated during the first epoch (epoch=0)?

### ISSUE THREAD

**Issue by yyun** [DulyHao/AlphaForge#13]
generater.py  lines 301，
if epoch>0:
            loss.backward()
            opt.step()

**Comment by DulyHao** [comment]
This is not important, just wanna see the performance difference between random weights and weights after training.
